node/tools/src/k8s.rs

```diff
@@ -63,6 +82,59 @@ pub async fn create_or_reuse_namespace(client: &Client, name: &str) -> anyhow::R
     }
 }
 
+pub async fn create_tests_deployment(client: &Client) -> anyhow::Result<()> {
+    let deployment: Deployment = serde_json::from_value(json!({
```

Comment:
Why hardcoding json? It should be feasible to construct a Deployment object directly, no?

Reply:
Yeah, I don't like the idea of hardcoded JSON either, but I was avoiding the other approach using the `kube` crate because it adds a lot of boilerplate, at least for this specific deployment. I changed it anyway since it's a pretty straightforward change. It adds a lot of `Default::default()` for the fields that aren't needed and a lot of `Options` for the optional fields. Maybe we can explore other options or decide on whichever you think is better.

Reply:
Nah, I didn't like the JSON either, so we can go with this solution. Also, I think it has more room for improvements or refactors in the future if needed.